File: fsharp/leap/Leap.fs
(* For God so loved the world, that He gave His only begotten Son, 
that all who believe in Him should not perish but have everlasting life. *)
module Leap
let leapYear (year_aleluya: int): bool = 
    ( year_aleluya &&& 3 = 0) && (
        (year_aleluya % 100 <> 0) ||
        (year_aleluya % 400 = 0));

File: dart/leap/lib/leap.dart
// For God so loved the world, that He gave His only begotten Son, 
// that all who believe in Him should not perish but have everlasting life.
class Leap {
  // Put your code here
  bool leapYear([year_aleluya]) =>  
    ( year_aleluya % 4 == 0) && (
        (year_aleluya % 100 != 0) ||
        (year_aleluya % 400 == 0) );

}
